_posts/static/2018-02-01-installation-wizard.md

```diff
@@ -49,73 +47,81 @@ Flyve MDM v2.0.0 runs on GLPI 9.2.1 or later. It depends on inventory features o
+* Unpack the file in:
 
-### Notifications
+```console
+/var/www/
+```
 
-* Login in GLPI with a super admin account.
-* Go to **Setup > Notifications**
-  * Enable followup
-  * Enable followups via email
+<img src="{{ '/images/picto-warning.png' | absolute_url }}" alt="Careful!" height="16px"> Keep in mind the type of file you download, to unpack a tarball you need the <a name="commands"></a>following command:
 
-<img src="{{ '/images/step-notifications.png' | absolute_url }}" alt="Enable Email Notifications">
+```console
+tar -xvzf file-name.tar.gz
+```
 
-Click on **Email followups configuration** and setup the form depending on your requirements to send emails. This step is available from the wizard.
+or
 
-### Enabling the Rest API
+```console
+tar -xvzf file-name.tgz
+```
 
-* Go to **Setup > General** and select the tab **API**
-  * Enable Rest API
-  * Enable **login with credentials**
-  * Enable **login with external tokens**
-  * Check there is a full access API client able to use the API from any IPv4 or IPv6 address (click it to read and/or edit)
+* Go to the glpi folder and **run composer install --no-dev**
 
-<img src="{{ '/images/enable-api.png' | absolute_url }}" alt="Enable API">
+Here you have the link to the [GLPI Project Wiki](http://wiki.glpi-project.org/doku.php?id=en:welcome), there you will find more information about GLPI.
 
-From the wizard you will be able to check this configuration is as it should.
+### Fusion Inventory plugin
 
-## Configuration of Flyve MDM for GLPI
+* Download FusionInventory 9.2+1.0 for GLPI, you can get it with any of these methods:
 
-These steps can be followed from the Installation Wizard.
+  * From [GitHub Releases](https://github.com/fusioninventory/fusioninventory-for-glpi/releases)
+  * Using Git:
 
-The Wizard will help you to check and complete all the mandatory steps in order to obtain a successful setup of Flyve MDM.
+    ```console
+      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git
+    ```
+* Go to glpi/plugins/ and unpack the file there
 
-### Cron Job
+<img src="{{ '/images/picto-warning.png' | absolute_url }}" alt="Careful!" height="16px"> Keep in mind the type of file you download, to unpack a tarball see the [commands listed above](#commands).
 
-Flyve MDM requires cron job to run Automatic actions
+Here is the link to the [Fusion Inventory Documentation](http://fusioninventory.org/documentation/).
 
-<img src="{{ '/images/step-cron.png' | absolute_url }}" alt="Cron Job">
+### Flyve MDM plugin
 
-### Email configuration
+* Download Flyve MDM for GLPI v2.0.0-dev, you can get it with any of these methods:
 
-Adapt the settings to your requirements
+<!--  * From [GitHub releases](https://github.com/flyve-mdm/glpi-plugin/releases)-->
+  * [Download zip file](https://github.com/flyve-mdm/glpi-plugin/archive/develop.zip) from GitHub (The zip is from the develop branch, which is the v2.0.0-dev)
+  * Using Git:
```

Comment:
IF we specify the version of the development branch, we need to update it each time we do a release. Will we think about  that each time we need ?

Reply:
I know I shouldn't but I'm doing it since we want to make sure the users get that version and they know exactly how to download it, especially since it is not in the release section yet (hence the commented line) , later I want to make more generic instructions for the plugin download same with FusionInventory (not specifying a version), so the only thing we need to update is the compatibility matrix with each release

Reply:
I prefer updating the matrix for each release. Maybe we should add a hyperlink to the matrix, targeting a new tab or window of the browser ?

By the way, we have probably most of version requirements in the source code. We might use that to automatically update the matrix... Just an idea.

Reply:
I don't understand the part of the hyperlink, to what would we be linking the matrix? Or do you refer to link each version to their installation documentation?

Reply:
NEvermind. You're right. The document only mentions the develop branch for now. Then leave this line as is.